project-base/storefront/components/Basic/Animations/AnimateCollapseMenu.tsx

```diff
@@ -0,0 +1,24 @@
+import { TIDs } from 'cypress/tids';
+import { HTMLMotionProps, m } from 'framer-motion';
+import { collapseExpandMenuAnimation } from 'utils/animations/animationVariants';
+
+export const AnimateCollapseMenu: FC<HTMLMotionProps<'div'> & { tid?: TIDs; keyName?: string }> = ({
+    children,
+    className,
+    keyName,
+    tid,
+    ...props
+}) => (
+    <m.div
+        key={keyName}
+        animate="open"
+        className={className}
+        exit="closed"
+        initial="closed"
+        tid={tid}
+        variants={collapseExpandMenuAnimation}
+        {...props}
+    >
+        {children}
+    </m.div>
+);
```

Comment:
I can still see a little bit of flickering. Maybe try different animation then `collapse`? Maybe try appearing animation (for example similar as in Breno)?

Reply:
changed to fade animation